app/src/main/assets/setup.properties

```diff
@@ -1,2 +1,3 @@
 #
-setup.admin_web_console=http://demo.flyve.org
\ No newline at end of file
+setup.admin_web_console=http://demo.flyve.org
+setup.thestralbot=http://demo.flyve.org
```

Comment:
https://hooks.thestralbot.com/-1001061475099/

Reply:
this is a fake url, the real url is in circleci